assets/js/app.js

```
@@ -1,264 +1,533 @@
-// Main site-wide JavaScript utilities
+// Global State
+let state = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'state' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)